npc/re/instances/GhostPalace.txt

```diff
@@ -0,0 +1,975 @@
+		unittalk $@mobid[0], "Sakray..?? why.....Sakray!";
+		monster 'map_spa$,201,64, "Tarnished Oath#45" ,2954,1, .@label$;	// COLORLESS_VOW
+		unittalk $@mobid[0], "Sakray killed the King and tried to kill us!";
+		monster 'map_spa$,175,44, "Fatal Days#43" ,2958,1, .@label$;		// FATAL_DAYS
+		unittalk $@mobid[0], "Sakray killed a princess !";
+	}
+	end;
+OnMyMobDead3:
+	if (mobcount('map_spa$, instance_npcname("#gp_control_6") + "::OnMyMobDead3") < 1) {
+		sleep 2000;
+		monster 'map_spa$,200,45, "Torturous Redeemer#21" ,2961,1, instance_npcname("#gp_control_6") + "::OnMyMobDead4";	// E_TORTUROUS_REDEEMER
+		unittalk $@mobid[0], "Thanatos Magic Trace! I will destroy you if I can't have you... Oh no! How could this be?";
+		sleep 5000;
+		killmonster 'map_spa$, instance_npcname("#gp_control_6") + "::OnMyMobDead4";
+		enablenpc instance_npcname("Sakray#dk");
+		enablenpc instance_npcname("Thanatos Magic Trace#dk3");
+		disablenpc instance_npcname("#gp_control_6");
+	}
+	end;
+OnMyMobDead4:
+	end;
+}
+
+1@spa,196,44,5	script	Sakray#dk	4_M_SAKRAY,{
+	mes "[Lurid Royal Guard]";
+	mes "Are you satisfied..? Thanatos Magic Trace?";
+	npctalk "Lurid Royal Guard: Are you satisfied..? Thanatos Magic Trace?";
+	next;
+	mes "[Thanatos Magic Trace]";
+	mes "I need more blood..Sakray, offer me his blood.";
+	npctalk "Thanatos Magic Trace: I need more blood..Sakray, offer me his blood.", instance_npcname("Thanatos Magic Trace#dk3");
+	next;
+	mes "[Sakray]";
+	mes "Okay, that is good. A traveller. Did you see?";
+	npctalk "Sakray: Okay, that is good. A traveller. Did you see?";
+	next;
+	mes "[" + strcharinfo(0) + "]";
+	mes "Sakray, What do you want sympathy? Or is it a trap?";
+	unittalk getcharid(3), "" + strcharinfo(0) + " : Sakray, What do you want sympathy? Or is it a trap?";
+	next;
+	mes "[Sakray]";
+	mes "None...I did not need to make an effort to catch a novice.";
+	npctalk "Sakray: None...I did not need to make an effort to catch a novice.";
+	next;
+	mes "[" + strcharinfo(0) + "]";
+	mes "If so... then why...?";
+	unittalk getcharid(3), "" + strcharinfo(0) + " : If so... then why...?";
+	next;
+	cutin "cry-b.bmp",2;
+	mes "[Sakray]";
+	mes "I wanted to get rid of a piece of humanity inside of me through you.";
+	npctalk "Sakray: I wanted to get rid of a piece of humanity inside of me through you.";
+	next;
+	mes "[Sakray]";
+	mes "I am quite satisfied with the result.";
+	next;
+	mes "[Thanatos Magic Trace]";
+	mes "Sakray, offer the blood to me now!";
+	npctalk "Thanatos Magic Trace: Sakray, offer the blood to me now!", instance_npcname("Thanatos Magic Trace#dk3");
+	next;
+	mes "[Sakray]";
+	mes "If you were me, what would you do?";
+	next;
+	mes "[Sakray]";
+	mes "Can you set yourself free from this curse?";
+	next;
+	mes "[" + strcharinfo(0) + "]";
+	mes "...";
+	unittalk getcharid(3), "" + strcharinfo(0) + " : ...";
+	next;
+	mes "[Sakray]";
+	mes "Heh heh... I will keep watching you until the day you grow up enough.";
```

Comment:
`Heh heh... I will keep watching you until the day comes when you are old enough.`